### ISSUE TITLE
Alternate label language for periods

### ISSUE THREAD

**Issue by ptgolden** [periodo/periodo-data#16]
Am I correct that it is always assumed that "alternate labels" will be in English?

which means:
  * `label` is how the period the period was defined in the source
  * `localizedLabel`is how the period was translated in the source (?)
  * `alternateLabels` are created by the curator (?)

I supposed I don't understand the distinction between localizedLabel and alternateLabels except that the latter hardcodes English and allows for multiple values.

**Comment by atomrab** [comment]
I'm not sure what column mapped to localizedLabel, but I think it goes something like this. 

The original label is "Early Bronze Age Period (Early Cypriote I)".

The localizedLabel is our editorial cleanup of the first part of that label, for display in the interface, always in English (or at least we called it label_en in the spreadsheet): "Early Bronze Age" (dropping the "Period", and moving the rest to the alternate label).

The alternateLabel is "Early Cypriote I", which we didn't make up, but which we separated from the original label which had an internal alternate (again, always in English, translated from the original label). 

We always did alternate labels in English in the spreadsheet, even when we'd pulled them out of a combined period term in another language. Consider this example:

Original term=label: "Mesjetë /Bizantine" (Albanian)
Our localizedLabel (label_en in the spreadsheet): "Medieval" (translation of Albanian "Mesjete")
Our alternateLabel (label_alt in the spreadsheet): "Byzantine" (translation of Albanian "Bizantine")

Is this enough for semantic consistency, or should we also provide the alternate term as a separate value in the original language, and then as an English translation? I'd prefer the current system for simplicity, but will it make search in other languages harder?

**Comment by ptgolden** [comment]
The localizedLabel is not always in English in the dataset. For example:
```
"id": "p0vn2fr2ft3",
"alternateLabel": [
  "Late Mesolithic"
],
"label": "Yngre mesolitikum",
"localizedLabel": {
  "eng-latn": "Late Mesolithic",
  "swe-latn": "Yngre mesolitikum"
},
...
```

**Comment by rybesh** [comment]
There is (purposefully) some redundancy here.

A period definition always has a `label` that is how it appeared in the source. Examples:
* `Early Bronze Age Period (Early Cypriote I)`
* `Mesjetë e hershme`

**If** the `label` is not English **or** it needs some cleanup/normalization, a period definition may be given one or more `alternateLabel`s. These are always English. Examples:
* Early Bronze Age III
* Early Cypriot III
* Early Medieval

The `localizedLabel` is simply there to assign language tags to the various labels. It should not contain any values that are not already a `label` or an `alternateLabel`.

Looking at the data, I see a few cases where there is a 2nd `alternateLabel` that does not appear in `localizedLabel`, which I guess is a data quality bug.

**Comment by ptgolden** [comment]
Does that mean that localizedLabel should be able to have multiple entries for a single language?

**Comment by ptgolden** [comment]
(i.e. a `localizedLabel` value for each different `alternateLabel`)?

**Comment by rybesh** [comment]
No. Now I remember why I only included at most one `alternateLabel` in `localizedLabel`, because the JSON-LD language indexing stuff will only work with one value per language tag. So it's fine. If they ad more than one `alternateLabel` just put the first one in `localizedLabel`.

**Comment by ptgolden** [comment]
My issues are resolved in #19.
